### DISCUSSION TITLE
Why does the behaviour  change when I attach an empty ExampleResourceRequestHandler

### DISCUSSION THREAD

**Discussion by okansarica** [cefsharp/CefSharp#3934] [General]
Hi,

I need to catch some network responses for the given URL  [https://www.just-eat.ch/speisekarte/restaurant-pumpkin-1. ](url)
I pulled the code from master and changed it a little bit. The change steps are below
1- Return new ExampleResourceRequestHandler(); from GetResourceRequestHandler in ExampleRequestHandler class for every URL (simply removed the if condition)
2- Commented all of the function overrides in ExampleResourceRequestHandler 

There are a few requests in the network tab to https://cw-api.takeaway.com  so after attaching the resource request handler I am getting 400 from these APIs. I need to learn what is changing when I attached a handler that causes the request to change

What is the difference between returning null or empty class from GetResourceRequestHandler ?

**Comment by amaitland** [comment]
> I pulled the code from master and changed it a little bit. The change steps are below

Rather than describe code changes, push them to GitHub and provide a link.

For testing purposes I'd suggest using https://github.com/cefsharp/CefSharp.MinimalExample which uses the latest stable version.
